pkg/assembler/backends/neo4j/src.go

```diff
@@ -411,6 +411,10 @@ func (c *neo4jClient) sourcesNamespace(ctx context.Context, sourceSpec *model.So
 	return result.([]*model.Source), nil
 }
 
+func (c *neo4jClient) IngestSources(ctx context.Context, sources []*model.SourceInputSpec) ([]*model.Source, error) {
+	panic("not implemented")
```

Comment:
for consistency, return err not implemented instead of panic

Reply:
changed